### ISSUE TITLE
Quick paste opens up the create link functionality in programs that have it

### ISSUE THREAD

**Issue by codeinaire** [IndieGoodies/PastePal#91]
For context I've been using the Paste Setapp program on a different mac and it has been brilliant and easy to use so I'm pretty much expecting that type of behaviour from PastePal. I paid for it just to be sure that I was getting all the functionality.

**Describe the bug**
I've set quick mode to use paste and I've setup a keyboard shortcut to toggle side window. When I open the side window with the shortcut and go to paste an item in an application it doesn't seem to paste. I've tried it in Evernote but instead of pasting it opens up the link creation functionality and it does the same for the Notes app in mac. The keyboard shortcut for that is command+k. When I tried it with Slack it doesn't paste anything.

**To Reproduce**
1. Set Quick mode to paste
2. Set toggle side window keyboard short cut to control+alt+k
3. Copy something and navigate to Evernote or Slack or the mac Notes app
4. Us the toggle window keyboard shortcut and try to paste an item into the Notes app
5. It opens up the link creation functionality

**Expected behavior**
It should paste the item that was selected instead of opening up the link creation functionality.

**Screenshots**

https://github.com/user-attachments/assets/a5249884-0b71-471d-b5a0-5802e4729307

**Desktop (please complete the following information):**
 - OS: sequoia 15.0.1
 - Browser firefox dev edition 132.0b9 (aarch64)

**Comment by onmyway133** [comment]
@codeinaire Hi John, thanks for letting me know. Let me investigate this a bit

**Comment by onmyway133** [comment]
@codeinaire Hi, I've checked a bit. For example Notes, the keyboard shortcut to create link is Cmd+K https://support.apple.com/guide/notes/keyboard-shortcuts-and-gestures-apd46c25187e/mac


When you said "Set toggle side window keyboard short cut to control+alt+k", do you mean `Control + Option + K`? since on Mac there is not Alt key. I tried with `Control + Option + K` with Quick Mode to paste to Notes app and it is working for me. I'm on macOS 15.0.1

<img width="744" alt="Screenshot 2024-10-25 at 16 27 18" src="https://github.com/user-attachments/assets/22302ef9-59ee-47a1-bb24-eceea8a99e75">

**Comment by codeinaire** [comment]
Yeah, I do mean `control + option + K`, I have a keyboard skin that also marks it as `alt`. 
<img width="697" alt="Screenshot 2024-10-30 at 8 12 55 am" src="https://github.com/user-attachments/assets/3e9a41f9-6dbe-4e52-ad45-1f27c6bd22b4">

I'm also on macOS 15.0.1 but I'm still having the issue. Do you have any recommended work-arounds?
